Add GitHub issue-nonpr badge

BaerMitUmlaut opened the pull request:
This adds a variant of the GitHub issue badge which only counts real issues and skips PRs. Issues normally counting pull requests as well doesn't seem intuitive, but I didn't want to change the default one to keep backwards compatibility. New syntax is:
```
 /github/issues-nonpr/user/repo.svg
```

Ref. #420

paulmelnikow commented:
Thanks for this contribution! Look great and would love to get it merged. Sorry for the delay!

Could I ask you to add an entry to `try.html`, and to rebase?

BaerMitUmlaut commented:
Rebased, fixed the conflict and added it to `try.html`, should be good to go now. I think Travis is failing because the current master fails?

What do you think about using this instead of the normal issue counter? I believe it's probably what people expect from the normal `issue` badge.

paulmelnikow commented:
> What do you think about using this instead of the normal issue counter? I believe it's probably what people expect from the normal `issue` badge.

I tend to agree. However issues and pull requests are combined into a combined search interface in the UI, so it's not a total surprise, and in the discussion at #420 @espadrine said he thought the current behavior should be preserved for `issues`.

@espadrine That was a couple years ago… do you still feel that way?

Perhaps we could update the text for that badge, to say "issues and pull requests"? That change would make it clear what they're getting, and could encourage users to consider the alternatives.

paulmelnikow commented:
@espadrine Could I ask you to respond to this question? ^^

Would love to get this merged.

vossad01 commented:
The current state of things seems to have both the **issues** badge and the **pull requests** badge showing the sum of issues _and_ pull requests.  I would argue that not fixing the current badges would violate the [shields specification](https://github.com/badges/shields/blob/master/spec/SPECIFICATION.md) because the number does not represent what is semantically indicated on the left-hand side.

Adding badges today, if I see the number is in correct, but I want the badge, I would still add it trusting the bug to be fixed and not expect to need to change it later.  However, if the current issues and PR badges are not to be fixed, I would say the text should be changed to says "issues and pull requests" so that it accurately represents what is displayed and consumers might notice and change it to the "fixed" issues badge if they want only issues.

paulmelnikow commented:
Merged via #1020.
